Chat/src/index.tsx

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './app/App';
+import { SwitchableFluentThemeProvider } from './app/theming/SwitchableFluentThemeProvider';
+
+if (document.getElementById('root') !== undefined) {
+  ReactDOM.render(
+    <SwitchableFluentThemeProvider scopeId="SampleChatApp">
+      <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0" />
```

Comment:
Why do we need this meta line? Can we add a comment so sample users know why they should or shouldn't include this

Reply:
Created ADO Item
https://skype.visualstudio.com/SPOOL/_workitems/edit/2691369